Doc/data/refcounts.dat

```diff
@@ -620,7 +620,9 @@ PyErr_GetExcInfo:PyObject**:pvalue:+1:
 PyErr_GetExcInfo:PyObject**:ptraceback:+1:
 
 PyErr_GetRaisedException:PyObject*::+1:
-PyErr_SetRaisedException::::
+
+PyErr_SetRaisedException:void:::
+PyErr_SetRaisedException:PyObject *:exc:0:stolen
```

Comment:
Oh wait, should this be -1? Also it should probably be consistent with pointer stars:
```suggestion
PyErr_SetRaisedException:PyObject*:exc:0:stolen
```

Reply:
We really need to have a different symbol for stolen references... (I suggest using $ for "stolen"). At the top of the file it says that:

```text
# XXX NOTE: the 0/+1/-1 refcount information for arguments is
# confusing!  Much more useful would be to indicate whether the
# function "steals" a reference to the argument or not.  Take for
# example PyList_SetItem(list, i, item).  This lists as a 0 change for
# both the list and the item arguments.  However, in fact it steals a
# reference to the item argument!
```

By the way, I'd like to convert in the future `T *` instead of `T*` because we also have entries of the form `void (*)(PyObject *)` so I think it's preferrable to have the same format everywhere.

Reply:
Ah, good to know. Thanks!

Reply:
Agreed about using `$` to indicate stolen references.

Reply:
I'll create a PR for that tomorrow as well.